### ISSUE TITLE
The latest Obsidian 1.6.3 update

### ISSUE THREAD

**Issue by Shuricus** [Richardsl/heatmap-calendar-obsidian#128]
After the latest Obsidian 1.6.3 update, it doesn't show heatmaps, but it doesn't give an error either.

**Comment by snowfield516** [comment]
Yes, the heatmap shows nothing.
![image](https://github.com/Richardsl/heatmap-calendar-obsidian/assets/4134369/a8d9df7d-735c-4b2d-902b-be2238055c85)

**Comment by ZhaoZhikangHub** [comment]
Yes, I have the same question
![Snipaste_2024-06-13_10-57-22](https://github.com/Richardsl/heatmap-calendar-obsidian/assets/48825102/7c561e5b-216e-40fc-b61b-5787f42663bd)

**Comment by DanielSRojo** [comment]
Exactly same issue over here. I reviewed console output, but I did not see anything relevant.
![image](https://github.com/Richardsl/heatmap-calendar-obsidian/assets/66113104/e414b5bd-add4-4555-85ba-4653a1323e0e)

**Comment by Emindu** [comment]
I have same issue.
![image](https://github.com/Richardsl/heatmap-calendar-obsidian/assets/41495728/c6ccce1d-8641-45c4-b110-839043b413a8)

**Comment by SergioAlias** [comment]
It looks like a generalized issue, same problem here. I'm not posting another screenshot but it only shows a tiny line for the current day.

**Comment by spannercodes** [comment]
Fixed this for myself by adding the following CSS to `.obsidian/plugins/heatmap-calendar/styles.css` in my vault:
```css
.heatmap-calendar-boxes>li {
  width: 100%;
}
```

**Comment by SergioAlias** [comment]
> Fixed this for myself by adding the following CSS to `.obsidian/plugins/heatmap-calendar/styles.css` in my vault:
> 
> ```css
> .heatmap-calendar-boxes>li {
>   width: 100%;
> }
> ```

Worked like a charm for me. Many thanks!

**Comment by magnatom** [comment]
Lol I though I had been doing something wrong.

**Comment by anandbhaskaran** [comment]
As this is one of the plugins that is used, let's aim to fix it soon. It is a pity that the plugin is in broken state for a while now

**Comment by Richardsl** [comment]
released new version with fix, please update and check if it fixes the issue

**Comment by bangsluke** [comment]
Yep - just updated to version 0.7.1 and working perfectly - thanks Richardsl!

**Comment by fetwar** [comment]
Can confirm, after upgrading to 0.7.1 the issue is resolved
